back/app/models/project.rb

```diff
@@ -167,10 +167,6 @@ def timeline?
     process_type == 'timeline'
   end
 
-  def native_survey?
```

Comment:
Why is this being removed in this PR?

Reply:
It's a duplicate of `native_survey?` defined on the participation context.